com/truckmuncher/api/trucks.proto

```diff
@@ -77,6 +85,19 @@ message ActiveTrucksResponse {
          * Value in the range [-180, 180]
          */
         required double longitude = 3;
+        /** 
+         * Indicates whether the truck's location is verified or not.
+         */
+        optional bool verified = 4;
+        /**
+         * Indicates the current upvote/downvote score.
```

Comment:
Are there bounds on this? Can it be negative? I don't remember what we decided but we should document it. Should the API return trucks that are negative at all in this request? If it does, should the clients display those trucks? What happens at a score of 0?